docs/odoo/models/mail.message.rst

```diff
@@ -20,14 +20,14 @@ It needs to be defined in XML which will have following syntax.
         <field name="description">Task planned hours changed</field>
     </record>
 
-Users can also have a mail.message.subtype that depends on an other to act through a relation field. For the planned hours, we can have following sysntax for it.
+Users can also have a mail.message.subtype that depends on an other to act through a relation field. For the planned hours, we can have following syntax for it.
 
 .. code-block:: xml
 
     <record id="mt_task_planned_hours_change" model="mail.message.subtype">
         <field name="name">Task planned hours changed</field>
         <field name="sequence">10</field>
-        <field name="res_model">project.task</field>
+        <field name="res_model">project.project</field>
```

Comment:
ok, parent_id misleaded me

    parent_id = fields.Many2one(
        'mail.message.subtype', string='Parent', ondelete='set null',
        help='Parent subtype, used for automatic subscription. This field is not '
             'correctly named. For example on a project, the parent_id of project '
             'subtypes refers to task-related subtypes.')